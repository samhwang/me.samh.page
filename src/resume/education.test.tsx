import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import Education from './education';

describe('Education section render', () => {
  it('Should match snapshot', () => {
    const EducationSection = (
      <BrowserRouter>
        <Education />
      </BrowserRouter>
    );
    const tree = render(EducationSection);
    expect(tree).toMatchSnapshot();
  });
});
