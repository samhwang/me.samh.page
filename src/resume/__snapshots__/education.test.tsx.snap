// Vitest Snapshot v1

exports[`Education section render > Should match snapshot 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        class="resume-section p-3 p-lg-5 d-flex align-items-center"
        id="education"
      >
        <div
          class="w-100"
        >
          <h2
            class="mb-5"
          >
            Education
          </h2>
          <div
            class="resume-item d-flex flex-column flex-md-row justify-content-between mb-5"
          >
            <div
              class="resume-content"
            >
              <h3
                class="mb-0"
              >
                Navitas Professionals
              </h3>
              <div
                class="subheading mb-3"
              >
                Professional Year Program in Computer Science
              </div>
              <div>
                incl. Cert IV in Business
              </div>
            </div>
            <div
              class="resume-date text-md-right"
            >
              <span
                class="text-primary"
              >
                Oct 2016 - Oct 2017
              </span>
            </div>
          </div>
          <div
            class="resume-item d-flex flex-column flex-md-row justify-content-between mb-5"
          >
            <div
              class="resume-content"
            >
              <h3
                class="mb-0"
              >
                RMIT University
              </h3>
              <div
                class="subheading mb-3"
              >
                Bachelor of Information Technology
              </div>
              <div />
            </div>
            <div
              class="resume-date text-md-right"
            >
              <span
                class="text-primary"
              >
                Oct 2012 - Dec 2015
              </span>
            </div>
          </div>
        </div>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      class="resume-section p-3 p-lg-5 d-flex align-items-center"
      id="education"
    >
      <div
        class="w-100"
      >
        <h2
          class="mb-5"
        >
          Education
        </h2>
        <div
          class="resume-item d-flex flex-column flex-md-row justify-content-between mb-5"
        >
          <div
            class="resume-content"
          >
            <h3
              class="mb-0"
            >
              Navitas Professionals
            </h3>
            <div
              class="subheading mb-3"
            >
              Professional Year Program in Computer Science
            </div>
            <div>
              incl. Cert IV in Business
            </div>
          </div>
          <div
            class="resume-date text-md-right"
          >
            <span
              class="text-primary"
            >
              Oct 2016 - Oct 2017
            </span>
          </div>
        </div>
        <div
          class="resume-item d-flex flex-column flex-md-row justify-content-between mb-5"
        >
          <div
            class="resume-content"
          >
            <h3
              class="mb-0"
            >
              RMIT University
            </h3>
            <div
              class="subheading mb-3"
            >
              Bachelor of Information Technology
            </div>
            <div />
          </div>
          <div
            class="resume-date text-md-right"
          >
            <span
              class="text-primary"
            >
              Oct 2012 - Dec 2015
            </span>
          </div>
        </div>
      </div>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
